### PR TITLE
[ui] fix for google analytics when loading in prod

### PR DESCRIPTION
**Pull request by thomasdavis**

### CODE REVIEW

File: ui/app/public/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=G-MSGFSQQEJ3"></script>
     <script>
-      if (window.location.href === "dex.sifchain.finance") {
+      if (window.location.host === "dex.sifchain.finance") {
```
Comment (banshee) [inline ui/app/public/index.html:12]: shouldn't this be hostname, not host?  (I thought host included :port - https://developer.mozilla.org/en-US/docs/Web/API/Location)

Comment (59023g) [reply ui/app/public/index.html:12]: `window.location.host` works. Go to dex.sifchain.finance console, then input `console.log(window.location.host)`. Will return desired `dex.sifchain.finance`

File: ui/app/src/components/tokenSelector/SelectTokenDialog.vue [hunk 2]
```diff
@@ -29,6 +29,7 @@ export default defineComponent({
     const searchText = ref("");
     const selectedTokens = props.selectedTokens || [];
 
+    const allTokens = actions.peg.getSifTokens()
```
Comment (59023g) [inline ui/app/src/components/tokenSelector/SelectTokenDialog.vue:32]: Please remove

**Review by 59023g — CHANGES_REQUESTED** [review]
This needs to be merged into develop. Also please remove `allTokens`

**Review by iverc — APPROVED** [review]

**Review by 59023g — APPROVED** [review]
Looks good

**Pull request merged**